core/src/serve_repair.rs

```diff
@@ -775,10 +777,12 @@ impl ServeRepair {
                 stats.err_unsigned += 1;
                 continue;
             }
-            if request.supports_signature()
-                && !Self::verify_signed_packet(&my_id, packet, &request, stats)
-            {
-                continue;
+            if request.supports_signature() {
+                if !Self::verify_signed_packet(&my_id, packet, &request, stats) {
+                    continue;
```

Comment:
I think would be good to also track number of signed requests which fail signature verification for sanity checks.
Also I assume that this will merge conflict with #27910 since we don't want to skip/continue here.

Reply:
This is already tracked by `err_sig_verify` and looks ok on testnet.

Reply:
ok, I think would be easier to review this once merge conflicts with #27910 are resolved.

Reply:
merged with #27910